packages/wxt/src/types.ts

```diff
@@ -535,6 +535,11 @@ export interface WxtDevServer
   restartBrowser: () => void;
 }
 
+export interface KeyboardShortcutWatcher {
+  start(): void;
+  stop(): void;
+}
+
```

Comment:
This file is for public types, so lets put the interface inside the new `src/core/keyboard-shortcuts.ts` file, next to `createKeyboardShortcuts`.

Reply:
Thanks for the clarifications and recommendations.
I'll quickly make those changes.